README.md

```diff
@@ -51,33 +51,34 @@ To see an example of an application using cryptofeed to aggregate and store cryp
 
 ## Supported exchanges
 
-* Binance
-* Binance US
-* Binance Jersey
-* Binance Futures
-* Bitcoin.com
-* Bitfinex
-* BitMax
-* BitMEX
-* Bitstamp
-* Bittrex
-* Blockchain
-* Bybit
-* Coinbase
-* Deribit
-* EXX
-* FTX
-* FTX US
-* Gemini
-* HitBTC
-* Huobi
-* HuobiDM
-* Kraken
-* Kraken Futures
-* OKCoin
-* OKEx
-* Poloniex
-* Upbit
```

Comment:
why was upbit removed?

Reply:
Good point.
To answer your question I suppose I dropped Upbit during my copy/past.
I remember having counted the number of exchanges before and after and found the same number...
OK I understand my mistake: I added Huobi Swap 🤦 
Thanks again for your review :-D

Note: I have also added the exchange names within the Pypi project keywords in `setup.py`:
https://github.com/olibre/cryptofeed/blob/clean-import-sections/setup.py#L45
The Upbit exchange was already there.